### DISCUSSION TITLE
Custom format function on DatePickerInput

### DISCUSSION THREAD

**Discussion by julamb** [mantinedev/mantine#4672] [Feature requests]
Let's say I want to have the day of the week in bold when displaying the selected value in DatePickerInput, and the rest of the date in normal weight.

Right now the only customization option we have seems to be the dayjs format string (with the `valueFormat` prop), it could be nice to have the option to provide a function that would take the current value as argument and return a ReactNode. Maybe a `renderValue` prop (like the `renderDay` prop on the DatePicker) ?

**Comment by rtivital** [comment]
See https://mantine.dev/dates/date-picker/#add-props-to-year-and-month-control
